taboos.json

```diff
@@ -388,5 +388,246 @@
+            },
+            {
+                "code": "03315",
+                "exceptional": 1,
+                "text": "This card gains the Exceptional keyword."
+            },
+            {
+                "code": "04309",
+                "text": "At the end of its ability, this card gains: “Remove All In from the game.”"
+            },
+            {
+                "code": "05116",
+                "text": "This card’s [action] ability is now a [fast] ability."
+            },
+            {
+                "code": "05188",
+                "text": "This card’s [action] ability is now a [fast] ability."
+            },
+            {
+                "code": "05189",
+                "text": "This card’s [action] ability is now a [fast] ability."
+            },
+            {
+                "code": "06195",
+                "text": "This card’s [action] ability now reads: “[action] Spend 1 ammo: <b>Fight.</b> You get +2 [combat] for this attack. If a chaos token with a non-negative modifier is revealed during this attack, this attack deals +2 damage.”"
+            },
+            {
+                "code": "02026",
+                "deck_limit": 0,
+                "text": "Forbidden."
+            },
+            {
+                "code": "02029",
+                "text": "This card now reads \"a symbol\" instead of the list of indicated symbols."
+            },
+            {
+                "code": "02269",
+                "text": "This card now reads \"a symbol\" instead of the list of indicated symbols."
+            },
+            {
+                "code": "60416",
+                "text": "This card now reads \"a symbol\" instead of the list of indicated symbols."
+            },
+            {
+                "code": "02263",
+                "text": "This card's fight ability now reads \"+1 damage\" instead of \"+2 damage.\""
+            },
+            {
+                "code": "03006",
+                "text": "This card's forced ability is moved to its reverse side, and it should now read: \"<b>Additional Setup</b>: After you draw your opening hand, choose a role ([survivor], [guardian], [seeker], [rogue], [mystic], or Neutral).\" This card's front side additionally gains: \"[action]: Switch roles. Does not provoke attacks of opportunity.\""
+            },
+            {
+                "code": "03019",
+                "text": "This card's revelation ability now reads: \"Discard 1 card in your hand or play area of your current role. Then...\""
+            },
+            {
+                "code": "04032",
+                "text": "This card now reads \"a non-[elder_sign] symbol\" instead of \"a [skull], [cultist], [tablet], [elder_thing], or [auto_fail] symbol.\""
+            },
+            {
+                "code": "04110",
+                "text": "This card now reads \"a non-[auto_fail] symbol\" instead of \"a [skull], [cultist], [tablet], or [elder_thing] chaos token.\""
```

Comment:
```suggestion
                "text": "This card now reads “a non-[auto_fail] symbol” instead of “a [skull], [cultist], [tablet], or [elder_thing] chaos token.”"
```